x-pack/test/performance/tests/playwright/ecommerce_dashboard.ts

```diff
@@ -5,52 +5,68 @@
  * 2.0.
  */
 import Url from 'url';
+import { Page } from 'playwright';
 import { FtrProviderContext } from '../../ftr_provider_context';
 
 export default function ecommerceDashboard({ getService }: FtrProviderContext) {
   describe('ecommerce_dashboard', () => {
-    const config = getService('config');
-    const performance = getService('performance');
-    const logger = getService('log');
+    it('ecommerce_dashboard', async () => {
+      const config = getService('config');
+      const performance = getService('performance');
+      const logger = getService('log');
 
-    const { step } = performance.makePage('ecommerce_dashboard');
+      await performance.runUserJourney(
+        'ecommerce_dashboard',
+        [
+          {
+            name: 'Go to Sample Data Page',
+            handler: async ({ page }: { page: Page }) => {
+              const kibanaUrl = Url.format({
+                protocol: config.get('servers.kibana.protocol'),
+                hostname: config.get('servers.kibana.hostname'),
+                port: config.get('servers.kibana.port'),
+              });
 
-    step('Go to Sample Data Page', async ({ page }) => {
-      const kibanaUrl = Url.format({
-        protocol: config.get('servers.kibana.protocol'),
-        hostname: config.get('servers.kibana.hostname'),
-        port: config.get('servers.kibana.port'),
-      });
+              await page.goto(`${kibanaUrl}/app/home#/tutorial_directory/sampleData`);
+              await page.waitForSelector('text="More ways to add data"');
+            },
+          },
+          {
+            name: 'Add Ecommerce Sample Data',
+            handler: async ({ page }: { page: Page }) => {
+              const removeButton = page.locator('[data-test-subj=removeSampleDataSetecommerce]');
+              try {
+                await removeButton.click({ timeout: 1_000 });
+              } catch (e) {
+                logger.info('Ecommerce data does not exist');
```

Comment:
Is `remove` operation a necessary step? If so, we should throw an exception. If not - maybe we should log with the `debug` level?

Reply:
It's a condition checks if server is already loaded ecommerce data, if there is data it tries to remove first and load again. I didn't understand why we would like to throw an exception if we want to remove 🤔